kaal-presentation/src/main/kotlin/cz/eman/kaal/presentation/lifecycle/channels/ChannelExt.kt

```diff
@@ -68,25 +69,27 @@ fun <A, B> combineLatest(
     var latestB: B? = null
     return scope.produce(context, capacity) {
         whileSelect {
-            channelA.onReceiveOrNull {
+            channelA.onReceiveCatching {
```

Comment:
Aren't channelA's and channelB's onReceiveCatching implementations the same? Shouldn't it to be written just once?